app-editors/helix/helix-23.03.ebuild

```diff
@@ -1,334 +0,0 @@
-# Copyright 2023-2023 Gentoo Authors
-# Distributed under the terms of the GNU General Public License v2
-
-# Auto-Generated by helix_ebuild.py (see FILESDIR)
```

Comment:
Please also remove this file with this commit.

Reply:
... and start to use pycargoebuild or similar.